### PR TITLE
Create noise app

### PR DESCRIPTION
**Pull request by sebschlo**

### CODE REVIEW

**Comment by sebschlo** [comment]
Current dependencies on/for this PR:
* master
  * **PR #3** <a href="https://app.graphite.dev/github/pr/breakfastny/flipdisc-python-framework/3" target="_blank">Graphite</a> 
    * **PR #4** <a href="https://app.graphite.dev/github/pr/breakfastny/flipdisc-python-framework/4" target="_blank">Graphite</a> 
      * **PR #5** <a href="https://app.graphite.dev/github/pr/breakfastny/flipdisc-python-framework/5" target="_blank">Graphite</a> 
        * **PR #6** <a href="https://app.graphite.dev/github/pr/breakfastny/flipdisc-python-framework/6" target="_blank">Graphite</a> 
          * **PR #7** <a href="https://app.graphite.dev/github/pr/breakfastny/flipdisc-python-framework/7" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/breakfastny/flipdisc-python-framework/7?utm_source=stack-comment).

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,6 @@ samples/**/*.zip
 *.egg-info/
 *.so
 *.pyc
+*.rdb
+flipdisc-core*
+binarize.c
```
Comment (sebschlo) [inline .gitignore:11]: these things seem to be all build artifacts and not source necessarily, what do you think about ignoring them?

Comment (wrigby) [reply .gitignore:11]: `binarize.c` is generated by Cython, which will generate different code depending on the Python version. The generated .c for Py2.7 used to be checked in, but I refactored the `setup.py` and removed it a few commits up this stack (see https://app.graphite.dev/github/pr/breakfastny/flipdisc-python-framework/3)

Publishing ti PyPI should be totally fine without the file checked in - builds from source will generate it themselves, and binary distributions will be unchanged.

Which is to say, I think it's probably a good thing to add `binarize.c` to the gitignore. Not sure about the .rdb thing, and I think the `flipdisc-core` stuff is from downloading the core distribution locally. Probably doesn't need to be ignored, but it won't really do harm either.

**Review by wrigby — APPROVED** [review]
[cool](https://app.graphite.dev/macros/breakfastny)

**Pull request closed without merging**